src/components/scenes/TransactionListScene.tsx

```diff
@@ -249,6 +251,7 @@ function TransactionListComponent(props: Props) {
     <SceneWrapper accentColor={iconColor} hasNotifications hasTabs>
       {({ insetStyles }) => (
         <>
+          <LinearGradient colors={[backgroundGradientColor, '#00000000']} start={{ x: 0, y: 0 }} end={{ x: 0, y: 1 }} style={StyleSheet.absoluteFill} />
```

Comment:
Hard coding black in to the gradient will not allow us to do light mode or whitelabels with different colors. Due to time constraints, I won't push back on this today but I'll create a task to add the gradient color to the theme. Ideally something like

`theme.txListBackgroundGradient` and `theme.txDetailsBackgroundGradient` which would have the full gradient capabilities of arbitrary direction and colors.

Reply:
Oh, good catch! We should use `theme.background.color for this one`.